packages/shell-api/src/mongo.ts

```diff
@@ -174,7 +176,7 @@ export default class Mongo extends ShellApiClass {
       const rc = this._serviceProvider.getReadConcern();
       return rc ? rc.level : undefined;
     } catch {
-      throw new MongoshInternalError('Error retrieving ReadConcern. Please file a JIRA ticket in the MONGOSH project');
+      throw new MongoshRuntimeError('Error retrieving ReadConcern.', CommonErrors.CommandFailed);
```

Comment:
This one isn't a runtime error, this is an internal error since this should never happen. If it does I think we have a drivers error.
```suggestion
      throw new MongoshInternalError('Error retrieving ReadConcern.', CommonErrors.CommandFailed);
```